config/routes.rb

```diff
@@ -1,17 +1,17 @@
 Rails.application.routes.draw do
-
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-  get '/api/v1/users', to: 'users#index'
-  get '/api/v1/users/:id', to: 'users#show'
-  post '/api/v1/users', to: 'users#create'
-  
+  get "/api/v1/users", to: "users#index"
+  get "/api/v1/users/:id", to: "users#show"
```

Comment:
I fixed this in what my new pull requests are already so maybe not necessary to do here?  but we are missing `api/v1/` namespacing for the items/users controllers, have to move them into the right folders, fix the controller file class title, and update in routes. takes like 60 seconds